core/src/main/java/com/datastrato/gravitino/catalog/TableEventDispatcher.java

```diff
@@ -50,12 +58,29 @@ public TableEventDispatcher(EventBus eventBus, TableDispatcher dispatcher) {
 
   @Override
   public NameIdentifier[] listTables(Namespace namespace) throws NoSuchSchemaException {
-    return dispatcher.listTables(namespace);
+    try {
+      NameIdentifier[] nameIdentifiers = dispatcher.listTables(namespace);
+      eventBus.dispatchEvent(new ListTableEvent(PrincipalUtils.getCurrentUserName(), namespace));
```

Comment:
Should you also add listed table names to the event?

Reply:
should we consider the scene that which there are too many table names?  and I wondered whether the table names are useful for list event.

Reply:
`listed table names` is the result data of the list operation, I think the listener only needs to care about the event action(and audit info), not the specific execution result data.

So I tend to not need to add listed table names to the event